buildbot/rotation.csv

```diff
@@ -202,7 +202,7 @@ Date,User
 2020-06-20,
 2020-06-21,
 2020-06-22,dibyo
-2020-06-23,christiewilson
+2020-06-23,jerop
```

Comment:
Since this is an override it might be worth adding a line to `rotation-overrides.csv` at the same time just to ensure this is honored if someone else regenerates the list between now and 2020-06-23.

Reply:
kk done!